### PR TITLE
[ALLUXIO-2601] Make MountOptionsTest final

### PR DESCRIPTION
**Pull request by exevvv**
https://alluxio.atlassian.net/browse/ALLUXIO-2601
To avoid accidental subclass extension

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

**Comment by exevvv** [comment]
alluxio-bot, check this please

File: pom.xml [hunk 1]
```diff
@@ -1182,3 +1182,5 @@
     </profile>
   </profiles>
 </project>
+
+
```
Comment (jsimsa) [inline pom.xml]: remove unnecessary empty line

**Comment by exevvv** [comment]
alluxio-bot, check this please

**Pull request merged**